### PR TITLE
Added about us page

### PR DESCRIPTION
**Pull request by gsalunke**
@Arpita-Jaiswal I have added about us page.
Task: https://app.asana.com/0/1204213004828720/1204214377232793/f
Please, review.

What changes are:
1. Added about-us components to `lib.ftd`
2. Added `common/commons.ftd` - we will remove `functions.ftd` once all other members raise their PRs using this file and changing current `functions.ftd` `import` from their files.
3. Updated `about.ftd` with dependencies.

### CODE REVIEW

File: lib.ftd [hunk 1]
```diff
@@ -2683,3 +2683,337 @@ link if { sneak-peak.cta-url != NULL }: $sneak-peak.cta-url
...
+$commons.team list members-list:
+
+-- ftd.column:
+width: fill-container
+align-content: center
+margin-top.px: 86
+
+-- ftd.column:
+width: fill-container
+max-width.fixed.px: 1320
```
Comment (Arpita-Jaiswal) [inline lib.ftd:2878]:
```suggestion
-- ftd.column:
width: fill-container
align-content: center
margin-top.px: 86
max-width.fixed.px: 1320
```

Why two columns?

File: lib.ftd [hunk 2]
```diff
@@ -2683,3 +2683,337 @@ link if { sneak-peak.cta-url != NULL }: $sneak-peak.cta-url
...
+$loop$: $our-team.members-list as $obj
+designation: $obj.designation
+avatar: $obj.avatar
+link: $obj.link
+
+-- end: ftd.column
+
+-- end: ftd.column
+
+-- end: ftd.column
```
Comment (Arpita-Jaiswal) [inline lib.ftd:2944]:
```suggestion
-- end: ftd.column
```

Why two columns as stated above?

**Review by Arpita-Jaiswal — CHANGES_REQUESTED** [review]

File: lib.ftd [hunk 3]
```diff
@@ -2683,3 +2683,337 @@ link if { sneak-peak.cta-url != NULL }: $sneak-peak.cta-url
...
+-- ftd.column:
+
+-- ftd.image:
+src: $assets.files.static.about-us.linkedin-green.svg
+width.fixed.px: 24
+height.fixed.px: 24
+link: $member-card.link
+align-self: start
+
+-- end: ftd.column
```
Comment (Arpita-Jaiswal) [inline lib.ftd:3011]:
```suggestion
-- ftd.image:
src: $assets.files.static.about-us.linkedin-green.svg
width.fixed.px: 24
height.fixed.px: 24
link: $member-card.link
align-self: start
```

File: lib.ftd [hunk 4]
```diff
@@ -2683,3 +2683,337 @@ link if { sneak-peak.cta-url != NULL }: $sneak-peak.cta-url
...
+
+-- ftd.image:
+if: { ftd.device != "mobile" }
+src: $member-card.avatar
+width.fixed.px: 376
+
+-- ftd.image:
+if: { ftd.device == "mobile" }
+src: $member-card.avatar
+width: fill-container
```
Comment (Arpita-Jaiswal) [inline lib.ftd:2982]:
```suggestion
-- ftd.image:
src: $member-card.avatar
width.fixed.px: 376
width if { ftd.device == "mobile" }: fill-container
```

File: lib.ftd [hunk 5]
```diff
@@ -2683,3 +2683,337 @@ link if { sneak-peak.cta-url != NULL }: $sneak-peak.cta-url
...
+-- ftd.column:
+width: fill-container
+min-width.fixed.percent if {ftd.device != "mobile"}: 33.33
+padding-horizontal.px if {ftd.device != "mobile"}: 12
+padding-horizontal.px if {ftd.device == "mobile"}: 20
+
+-- ftd.column:
+width: fill-container
+padding.px: 20 
+background.solid: $inherited.colors.background.step-1
```
Comment (Arpita-Jaiswal) [inline lib.ftd:2972]:
```suggestion
-- ftd.column:
width: fill-container
min-width.fixed.percent if {ftd.device != "mobile"}: 33.33
padding-horizontal.px if {ftd.device != "mobile"}: 32
padding-horizontal.px if {ftd.device == "mobile"}: 40
padding-vertical.px: 20
background.solid: $inherited.colors.background.step-1
```

**Review by Arpita-Jaiswal — CHANGES_REQUESTED** [review]

File: lib.ftd (hunk 5 above)
Comment (gsalunke) [reply lib.ftd]: @Arpita-Jaiswal two columns are required here as I tried to merge them the UI is breaking.
Basically UI is like this: https://admint.io/#/about

File: lib.ftd [hunk 6]
```diff
@@ -2064,7 +2064,7 @@ padding-horizontal: 41
 padding-vertical: 11
 marginTop: 2
 text-color: white
-$on-click$: $functions.formValidate()
+$on-click$: $common.formValidate()
```
Comment (Arpita-Jaiswal) [inline lib.ftd:2067]:
```suggestion
$on-click$: $common.form-validate()
```

**Review by Arpita-Jaiswal — CHANGES_REQUESTED** [review]

**Review by Arpita-Jaiswal — APPROVED** [review]

**Pull request merged**